internal/config/node/fuse.go

```diff
@@ -0,0 +1,35 @@
+package node
+
+import (
+	"bufio"
+	"os"
+	"strings"
+	"sync"
+)
+
+var (
+	hasFuseOnce sync.Once
+	hasFuse     bool
+	hasFuseErr  error
+)
+
+func HasFuse() bool {
+	hasFuseOnce.Do(func() {
+		f, err := os.Open("/proc/mounts")
+		if err != nil {
+			hasFuseErr = err
+			return
+		}
+		defer f.Close()
+
+		s := bufio.NewScanner(f)
+
+		for s.Scan() {
+			if strings.Contains(s.Text(), "/dev/fuse") {
```

Comment:
Why can't we stat `/dev/fuse` instead of parsing `/proc/mounts`?

Reply:
I googled "best way to find that a device is mounted" and that was the answer :laughing:

Reply:
That isn't needed :) You can use the `os` package to Stat the file, check if it is a device node and that should be sufficient.